tests/HHClientLinterTest.hack

```diff
@@ -0,0 +1,56 @@
+/*
+ *  Copyright (c) 2017-present, Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed under the MIT license found in the
+ *  LICENSE file in the root directory of this source tree.
+ *
+ */
+
+namespace Facebook\HHAST;
+use namespace HH\Lib\Str;
+
+final class HHClientLinterTest extends TestCase {
+  use LinterTestTrait;
+
+  /**
+   * The temporary directory to include testing source files to lint.
+   *
+   * Note that the temporary directory must be under the current directory,
+   * otherwise hh_client will not work.
+   */
+  const TMP_DIR = './.HHClientLinterTest.tmp';
```

Comment:
`__DIR__ . '/../.var/tmp/hhast/HHClientLinterTest'` for consistency with other temp files used by hhast, and so that they're already in .gitignore